new/LIT7416QeneLazeMaya.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7416QeneLazeMaya" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለዝ፡ ማየ፡ ዘለፉ፡ ዘበቈለ፡ ጥቀ፡ በአሰክቲነ፡ ንዑ፡ ንሥርዎ፡ ይእዜ፡ እምስረዊሁ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">La-zǝ māya zallafu za-baqʷala ṭǝqqa ba-ʾasakti-na nǝʿu nǝśrǝwo 
+                    ʾǝm-śǝrawi-hu (Qǝne of the śǝllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">For this water, they disproved what grew a lot in our testicles. See now, we are exterminating it
```

Comment:
This water of refutation, which emerged a lot from our testicles, come let us now exterminate it from its roots…